Sources/Load.php

```diff
@@ -951,11 +957,12 @@ function loadBoard()
 				LEFT JOIN {db_prefix}moderators AS mods ON (mods.id_board = {raw:board_link})
 				LEFT JOIN {db_prefix}members AS mem ON (mem.id_member = mods.id_member)
 			WHERE b.id_board = {raw:board_link}',
-			array(
+			array_merge(array(
 				'current_topic' => $topic,
```

Comment:
Might be good to declare $custom_column_parameters with both keys current_topic and board_link so mod authors can make use of them, even change them if they want to.

Reply:
Done